firmware/build/backpack/ws2812.cpp

```diff
@@ -261,6 +281,11 @@ void process_command(byte argc, byte *argv){
                         case PIXEL_COLOUR_BRG:
                             setColorOrderBRG();
                             break;
+                        case PIXEL_COLOUR_RGBW:
+                            isRGBW = true;
+                            color_depth = 4;
```

Comment:
@ajfisher I'm trying to make the value `color_depth` changable on the fly from 3 to 4. This doesn't seem to be working for me. Can you give me some guidance as to where I can change this value when switching to RGBW?